auto-deploy/deploy-haku.sh

```diff
@@ -30,6 +30,8 @@ for file in ${required_source_files}; do
 	fi
 done
 
+transfer_file_list+=" src"
+
```

Comment:
Github will not allow me to make comments on other parts of the code, but you need to add a -r to the scp command on line 39.

```
scp -i ${deployment_key} -r ${transfer_file_list} ${deployment_user}@${deployment_host}:${deployment_path}
```

In the future I will clean up deploy scripts to make it less dependent on file placement.